service/frontend/service.go

```diff
@@ -45,9 +45,10 @@ type Config struct {
 	EnableVisibilitySampling        dynamicconfig.BoolPropertyFn
 	EnableReadFromClosedExecutionV2 dynamicconfig.BoolPropertyFn
 	// deprecated: never used for ratelimiting, only sampling-based failure injection, and only on database-based visibility
-	VisibilityListMaxQPS          dynamicconfig.IntPropertyFnWithDomainFilter
-	EnableReadVisibilityFromES    dynamicconfig.BoolPropertyFnWithDomainFilter
-	EnableReadVisibilityFromPinot dynamicconfig.BoolPropertyFnWithDomainFilter
+	VisibilityListMaxQPS            dynamicconfig.IntPropertyFnWithDomainFilter
+	EnableReadVisibilityFromES      dynamicconfig.BoolPropertyFnWithDomainFilter
+	EnableReadVisibilityFromPinot   dynamicconfig.BoolPropertyFnWithDomainFilter
+	EnableLogCustomerQueryParameter dynamicconfig.BoolPropertyFn
```

Comment:
It is better to use BoolPropertyFnWithDomainFilter, so we can turn on/off this for some specific domains, for example we can turn this off for system domains